pt-BR/mail/chrome/messenger-smime/msgSecurityInfo.properties

```diff
@@ -37,33 +37,33 @@ EIInvalidLabel=A mensagem não pode ser descriptografada.
 EIInvalidHeader=Esta mensagem foi criptografada antes de ser enviada a você, mas não pode ser descriptografada.
 
 EIContentAltered=O conteúdo da mensagem aparenta ter sido modificado durante a transmissão.
-EINoDecryptCert=The certificate used to encrypt the message cannot be found.
-EIPasswordError=You did not enter your Master Password correctly.
-EIInvalidCipher=The message was encrypted using an encryption strength that this version of your software does not support.
+EINoDecryptCert=O certificado usado para criptografar a mensagem não pode ser encontrado.
+EIPasswordError=Você não digitou a senha mestra corretamente.
+EIInvalidCipher=A mensagem foi criptografada usando um tipo de criptografia que o seu aplicativo não suporta.
 EIClueless=Há problemas desconhecidos com esta mensagem criptografada.
 
 ## Signing Power Information string
-SPCanLabel=Message Can Be Signed
-SPCanHeader=You have chosen to digitally sign this message before sending it.
-SPCan=When other people receive your signed message, they can verify that the message comes from you and that it has not been altered since you signed it.
+SPCanLabel=A mensagem pode ser assinada
+SPCanHeader=Você escolheu assinar digitalmente esta mensagem antes de enviá-la.
+SPCan=Quando outras pessoas receberem sua mensagem assinada, elas podem verificar que a mensagem foi enviada por você e que ela não foi alterada desde que foi assinada.
 
-SPCannotLabel=Message Cannot Be Signed
-SPCannotHeader=You cannot digitally sign this message.
+SPCannotLabel=A mensagem não pode ser assinada.
+SPCannotHeader=Você não pode assinar digitalmente esta mensagem.
 
-SPNoCert=You have not specified a valid certificate for creating digital signatures.
-SPNoSigCert=The certificate you have specified for signing messages cannot be used for that purpose.
-SPNoValidCert=The certificate you have specified for signing messages is not yet valid. Make sure your computer's clock is set correctly.
-SPExpiredCert=The certificate you have specified for signing messages has expired.
-SPRevokedCert=The certificate you have specified for signing messages has been revoked.
+SPNoCert=Você não especificou um certificado válido para criar assinaturas digitais.
+SPNoSigCert=O certificado que você especificou para assinatura de mensagens não pode ser usado para este propósito.
+SPNoValidCert=O certificado que você especificou para assinatura de mensagens ainda não é valido. Certifique-se de que o relógio do seu computador esteja correto.
```

Comment:
CORRECTIONS NEEDED it should be SPNoValidCert=O certificado que você especificou para assinatura de mensagens ainda não é valido. Certifique-se de que o relógio do seu computador está correto.